CONTRIBUTING.md

```diff
@@ -66,35 +66,33 @@ Or to test a single feature:
+#### What qualifies as a backward incompatible change?
 
-Our initial commitment will be to command compatibility and parameter compatibility. However, since on the command line STDOUT and STDERR are essentially APIs we will make a best effort to keep machine-readable output compatibility, meaning if your code interfaces with Terminus via --json or --bash formats we will try our best to ensure these are stable and compatible between minor release. However, changes to the STDOUT, like success and fail messages, will not be consider incompatible. 
+Our initial commitment is to command compatibility and parameter compatibility. However, since on the command line STDOUT and STDERR are essentially APIs, we will make a best effort to keep machine-readable output compatibility. If your code interfaces with Terminus via --json or --bash formats, we will try our best to ensure these are stable and compatible between minor release. However, changes to the STDOUT, like success and fail messages, are not considered incompatible. 
 
-### Version branches 
+### Version Branches 
 
-If you are using terminus in a production environment you should be deploying the executable for the latest release. ( github.com/pantheon-systems/cli/releases )
+If you are using Terminus in a production environment, you should be deploying the executable for the latest release. ( github.com/pantheon-systems/cli/releases )
 
 Ongoing development on the next planned release will be on the master branch and should not be considered stable as changes will be taking place on a daily basis. 
 
-We will maintain a separate branch for all minor point releases going forward, i.e. 1.0.x, 0.5.x. Any critical patches should be submitted against those branches. If the fix is applicable to master a separate pull request can be made.
+We will maintain a separate branch for all minor point releases going forward, e.g. 1.0.x, 0.5.x. Submit any critical patches to those branches. If the fix is applicable to master, make a a separate pull request.
 
-### What this means for users ?
+#### What does this mean for users?
 
-0.5.0 will include only changes that are backward incompatible. After it's released we will create a 0.5.x branch that will be used for any critical bugs or patches that need to be addressed. All other bugs/features/issues will be addressed in the next major point  release 1.0.0. The master branch will effectively become the 1.0.x branch. There is currently no plan for a version 0.6.x. 
+0.5.0 will include only changes that are backward incompatible. After it's released, we will create a 0.5.x branch that will be used for any critical bugs or patches that need to be addressed. All other bugs/features/issues will be addressed in the next major point release 1.0.0. The master branch will effectively become the 1.0.x branch. There is currently no plan for a version 0.6.x. 
 
-
-Finally...
 ----------
 
 Thanks! Hacking on Terminus should be fun. If you find any of this hard to figure
```

Comment:
-Need to remove line break at end.
-Not an objective problem, but can we not tell people to "hack on" Terminus? Can we say "write for"?